src/components/swapv2/LimitOrder/useBaseTradeInfo.ts

```diff
@@ -36,25 +33,27 @@ export default function useBaseTradeInfo(currencyIn: Currency | undefined, curre
         })}`
       : null
   }
+
+  const controller = useRef(new AbortController())
   const fetchData = async (url: string | null): Promise<BaseTradeInfo | undefined> => {
     if (!currencyOut || !currencyIn || !url) return
-    const { data } = await axios.get(url, {
+
+    controller.current.abort()
+    controller.current = new AbortController()
+    const data = await fetch(url, {
+      signal: controller.current.signal,
       headers: {
         'X-Request-Id': sentryRequestId,
       },
-    })
-    const toCurrencyAmount = function (value: string, currency: Currency): CurrencyAmount<Currency> {
-      return TokenAmount.fromRawAmount(currency, JSBI.BigInt(value))
-    }
-    const { outputAmount, amountInUsd, amountOutUsd, routerAddress } = data
-    const amountOut = toCurrencyAmount(outputAmount, currencyOut)
+    }).then(data => data.json())
+
+    const { outputAmount, amountInUsd } = data
```

Comment:
May crash. Should wrap a try catch?

Reply:
no need, already added try catch in useSWR